.github/workflows/run_tests/action.yml

```diff
@@ -13,19 +13,18 @@ runs:
           ./tests/test_S_OPT
           mpirun -n 4 --oversubscribe tests/test_S_OPT
           ./tests/test_IncrementalSVD
-          ./tests/test_RandomizedSVD
           mpirun -n 3 --oversubscribe tests/test_RandomizedSVD
           ./tests/test_DMD
           mpirun -n 3 --oversubscribe tests/test_DMD
           ./tests/test_GreedyCustomSampler
           mpirun -n 3 --oversubscribe tests/test_GreedyCustomSampler
-
+# ./tests/test_RandomizedSVD
```

Comment:
Do we want to resolve issue #193 in this PR or open up a new PR?

Reply:
I think #193 is a separate issue so I propose to open up a new PR.